prediction/src/tests/test_grt123_preprocess.py

```diff
@@ -0,0 +1,169 @@
+import warnings
+
+import SimpleITK as sitk
+import numpy as np
+import pytest
+from scipy.ndimage import zoom
+from src.preprocess import load_ct, preprocess_ct, crop_patches
+
+"""
+Preprocessing tools used by the gtr123_models
+Code adapted from https://github.com/lfz/DSB2017
+"""
+
+
+class SimpleCrop(object):
+    """ """
+
+    def __init__(self):
+        self.crop_size = [96, 96, 96]
+        self.scaleLim = [0.85, 1.15]
+        self.radiusLim = [6, 100]
+        self.stride = 4
+        self.filling_value = 160
+
+    def __call__(self, imgs, target):
```

Comment:
Kinda gross to abuse dunderscore call. I'm assuming this code is from elsewhere?

Reply:
Yes, this code had been taken from the [edited version](https://github.com/concept-to-clinic/concept-to-clinic/commit/45db7be01d0ebcd4a91a071b4055eee4aac949cf#diff-af6ce34696667e4dea35064c3e319127R180) of the original [SimpleCrop](https://github.com/lfz/DSB2017/blob/76c6d60831a7d94b01e3774ed449acd78a525b78/training/classifier/data_classifier.py#L110) class. I've put unchanged functions into [`test_grt123_preprocess.py`](https://github.com/concept-to-clinic/concept-to-clinic/pull/170/files/389824f6042a5cbc4cb537c531ae40446ab6164a#diff-0ea94b8ddb8cba945d0d72ca74d13fd3) in order to ensure absolute interchangeability.